app/screens/MyCoursesScreen.tsx

```diff
@@ -18,29 +18,41 @@ const renderSeparator = () => {
 
 export const MyCoursesScreen: FC<MyCoursesScreenProps> = observer(function MyCoursesScreen() {
   // const { userStore } = useStores()
+  const scrollY = useRef(new Animated.Value(0)).current
   const username = undefined
 
   return (
     <Screen contentContainerStyle={$root} safeAreaEdges={["top"]}>
-      <View>
-        {/* Add user avatar here */}
-        <Text preset="subheading" style={$greeting} text={`Hi, ${username ?? "Hustler"}`} />
-      </View>
-      <Text preset="heading" style={$heading} text="My Courses" />
-      <TextField placeholder="Search videos..." />
-      <ListView
-        // keyExtractor={(item) => item.id}
-        contentContainerStyle={$listContainer}
-        data={[{}, {}, {}, {}, {}]}
-        estimatedItemSize={10}
-        ItemSeparatorComponent={renderSeparator}
-        renderItem={renderCourses}
-        showsVerticalScrollIndicator={false}
-      />
+      <ScrollView
```

Comment:
Instead of using a scrollview, try using the ListView StickyHeaderComponent prop for this feat, its a good feature to have.